core/domain/exp_domain_test.py

```diff
@@ -333,7 +333,7 @@ def test_validation(self):
 
         outcome.dest = destination
 
-        outcome.feedback = state_domain.SubtitledHtml('feedback_1', {})
+        outcome.feedback = state_domain.SubtitledHtml('feedback_1', '')
```

Comment:
@DubeySandeep could you please check whether this is correct? (Ditto elsewhere.)

Reply:
@siddhantkhandelwal, This should be a string instead of a dict. (changes are correct) 

Earlier we had audio_tarnslation attribute in the SubtitledHtml class which was a dict.

This incorrect values didn't get caught as this exploration is already falling validation check and it failed before reaching up to the level where we are validating the feedbacks/subtitleHtml. 

@siddhantkhandelwal, Thanks for fixing this!

Reply:
Thanks @seanlip @DubeySandeep